cli/cli.go

```diff
@@ -120,26 +138,53 @@ func GetCommands(logger utils.Log) []*clitool.Command {
 				if err != nil {
 					return
 				}
-				return printBuild(bld)
+				return printBuild(bld, context.String(formatFlag))
 			},
 		},
 	}
 }
 
-func printBuild(bld *build.Build) error {
+func printBuild(bld *build.Build, format string) error {
 	buildInfo, err := bld.ToBuildInfo()
 	if err != nil {
 		return err
 	}
-	b, err := json.Marshal(buildInfo)
-	if err != nil {
-		return err
-	}
-	var content bytes.Buffer
-	err = json.Indent(&content, b, "", "  ")
-	if err != nil {
-		return err
+
+	switch format {
+	case "cdx/xml":
```

Comment:
Let's use consts for the two cyclondx flags. The cosnsts should also be used as part of the ```format``` flag description.